__tests__/lint/enforcements-all-rules-warn/.redocly.yaml

```diff
@@ -0,0 +1,34 @@
+apiDefinitions:
+  main: ./openapi.yaml
+
+lint:
+  rules:
+    enforcements:
+      - on:
+          - PathItem.Operation.summary
+        description: Operation summary example warn
```

Comment:
Is the description a required property? What happens if it is not filled?

Reply:
Good question! @ohorbachevskyi make sure we have some good default description (ideally based on the failed enforcement)

Reply:
Added this one default description: "The assertion under <RULE_NAME> rule doesn't meet required conditions"

Reply:
Added next one default description:
"The assertion under '${rule.name}' rule doesn't meet required conditions"